snakePipes/shared/rscripts/DESeq2Report.Rmd

```diff
@@ -218,16 +218,20 @@ This plot shows how samples are clustered based on their euclidean distance usin
 ```{r PCA, eval = TRUE}
     # 5. PCA plots
     PCAdata$name<-make.names(PCAdata$name,unique=TRUE)
-    print(ggplot(PCAdata, aes(PC1, PC2, color = condition, shape = name)) +
+    if(nrow(PCAdata)>36){
```

Comment:
Do we inform this hardcoded value in the docs or CLI help message?

Reply:
To be honest, I have no idea what the rationale behind this limit is.  I just added a workaround to prevent the report for failing for users with those many samples. I guess it doesn't make sense to have 36 samples represented by 36 different shapes on a PCA plot - but what would be a good limit of readability? 10-15 shapes?
We don't have it documented, but that's a good idea to add it to the docs.

Reply:
I have added a line to the docs.